doc/usr/source/9_other/10_inductive_validity_core.rst

```diff
@@ -0,0 +1,94 @@
+.. _9_other/10_inductive_validity_core:
+
+Inductive Validity Core
+=======================
+
+The inductive validity core generation is a post-analysis treatement that computes a minimal subset of
+the model elements (contract items, equations or node calls) such that the system is still safe.
```

Comment:
contract items = assumptions, guarantees,
such that the system is still safe = that are sufficient to prove all properties